src/main.rs

```diff
@@ -0,0 +1,7 @@
+extern crate bellman;
+
+fn main(){
```

Comment:
please move this to an example instead, this is still a library :)

Reply:
Done